packages/wonder-blocks-icon/src/types.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * All the possible icon weights.
+ */
+export type PhosphorIconAsset = PhosphorRegular | PhosphorBold | PhosphorFill;
+/**
+ * The different icon weights per size.
+ */
+export type PhosphorIconSmall = PhosphorBold | PhosphorFill;
+export type PhosphorIconMedium = PhosphorRegular | PhosphorFill;
```

Comment:
**nit:** IDE's will label `PhosporIconSmall` with this comment, and then have no comment for `PhosphorIconMedium`. Give each its own comment.
```suggestion
/**
 * The different icon weights for small icons.
 */
export type PhosphorIconSmall = PhosphorBold | PhosphorFill;
/**
 * The different icon weights for medium icons.
 */
export type PhosphorIconMedium = PhosphorRegular | PhosphorFill;
```

Reply:
ah great point! will make the change.